src/server/sv_main.c

```diff
@@ -35,7 +35,7 @@ client_t *sv_client; /* current client */
 cvar_t *sv_paused;
 cvar_t *sv_timedemo;
 cvar_t *sv_enforcetime;
-cvar_t *timeout; /* seconds without any message */
+cvar_t *q2timeout; /* seconds without any message */
```

Comment:
why?

Reply:
I forgot about that - yeah the name "timeout" now conflicts with a variable in the curses.h file that the macOS 10.13 SDK includes. I'm not sure the path it's being included in, or if a makefile-based build has the same issue (probably not) so between this and the cmake thing you probably want to decline this pull request (however that works, not sure)